lte/gateway/Makefile

```diff
@@ -67,6 +67,8 @@ all: build
 
 build: build_python build_common build_oai build_sctpd build_session_manager ## Build all
 
+smf_build: build_session_manager  ## Build only sessionD component make smf_build
```

Comment:
what is the difference between this and running `make build_session_manager`?

Reply:
both are same, too much to type every time.  short hand reference added.